docs/getting-started.md

```diff
@@ -107,49 +107,54 @@ EOF
 ```
 
-**Note**: this is just an example of how it works. For further details you can visit [Postman documentation](executor-postman.md)
+**Note**: This is just an example of how it works. For further details you can visit the [Postman documentation](executor-postman.md).
 
-### Creating Cypress tests
+### **Creating Cypress Tests**
 
-Cypress tests are little more complicated to pass - for now we're supporting Git based paths for Cypress projects.
+Cypress tests are little more complicated to pass. TestKube currently supports Git based paths for Cypress projects.
 
-You can create new test with kubectl testkube plugin:
+You can create new tests with kubectl testkube plugin:
 
 ```sh
  kubectl testkube scripts create --uri https://github.com/kubeshop/testkube-executor-cypress.git --git-branch jacek/feature/git-checkout --git-path examples --name test-name --type cypress/project
 ```
 
 Where:
 
-- `uri` is git uri where testkube will get cypress project
-- `git-branch` is what branch should he checkout (default - main branch will be used)
-- `git-path` is what path of repository should be checked out (testkube is doing partial git checkout so it'll be fast even for very big monorepos)
-- `name` - is unique Sript Custom Resource name.
-- `type` - cypress/project - for Cypress based project test structure
+- `uri` is the git uri where testkube will get the cypress project.
+- `git-branch` is which branch should be checked out. By default the main branch will be used.
+- `git-path` denotes which path of the repository should be checked out. TestKube is doing a partial git checkout so it should be fast, even for very big monorepos.
+- `name` is the unique Script Custom Resource name.
+- `type` is cypress/project for a Cypress based project test structure.
 
-For now we're supporting only Cypress test runs, but we plan to fully integrate it.
+For now we're supporting only Cypress test runs, but plan to fully integrate it.
 
-## Starting new script execution
+## **Starting a New Script Execution**
 
-When our script is defined as CR we can now run it:
+When our script is defined as CR, we can now run it:
 
 ```shell
 kubectl testkube scripts start my-test-name 
 
 ... some script run data ...
 
-Use following command to get script execution details:
+```
+
+Use the following command to get script execution details:
+
+```shell
 kubectl testkube scripts execution 611b6da38cd74034e7c9d408
+```
 
-or watch for completition with
-kubectl testkube scripts watch 611b6da38cd74034e7c9d408
+Watch for completion with:
 
+```shell
+kubectl testkube scripts watch 611b6da38cd74034e7c9d408
 ```
 
-## Getting execution details
+## **Getting Execution Details**
 
-After script completed with success or error you can go back to script details by running
-scripts execution command:
+After a script completes with successfully or in error, run the scripts execution command to go back to the details:
```

Comment:
Instead of "After a script completes with successfully or in error..." 
it could be more like, "after the execution is complete, you can run go check its result: 

what do you think ? @exu